src/multipart.jl

```diff
@@ -69,7 +69,7 @@ function Form(d::Dict)
             io = IOBuffer()
         else
             write(io, "\r\n\r\n")
-            write(io, escapeuri(v))
```

Comment:
The `escape` comes from this commit https://github.com/JuliaWeb/HTTP.jl/commit/014d9f56363429568e12545f8d74fca93cc664cc#diff-1f61f32925620a8c37482af6f9a5d108R349.
As you say (#313), it doesn't seem to be called for by the RFC.
@quinnj do you recall the intention here?

Reply:
No, I don't recall; very well could be wrong.